pkg/controlplane/instance.go

```diff
@@ -579,15 +584,15 @@ func (m *Instance) InstallAPIs(apiResourceConfigSource serverstorage.APIResource
 	for _, restStorageBuilder := range restStorageProviders {
 		groupName := restStorageBuilder.GroupName()
 		if !apiResourceConfigSource.AnyVersionForGroupEnabled(groupName) {
-			klog.V(1).Infof("Skipping disabled API group %q.", groupName)
+			klog.V(1).InfoS("Skipping disabled API group.", "groupName", groupName)
```

Comment:
Maybe the key "apiGroup" would be more clear? Or at least keep it consistent.

Reply:
Fixed.